var/spack/repos/builtin/packages/lua-lpeg/package.py

```diff
@@ -11,12 +11,13 @@ class LuaLpeg(Package):
     """pattern-matching for lua"""
 
     homepage = "http://www.inf.puc-rio.br/~roberto/lpeg/"
-    url      = "https://luarocks.org/manifests/luarocks/lpeg-0.12-1.src.rock"
+    url      = "https://luarocks.org/manifests/gvvaughan/lpeg-1.0.2-1.src.rock"
```

Comment:
Why the URL change? Is the checksum of the older version still the same?

Reply:
> 
> 
> Why the URL change? Is the checksum of the older version still the same?

current URL is accessed only 0.12-1 or before: 
https://luarocks.org/modules/luarocks/lpeg
But new one is accessed newer version: 
https://luarocks.org/modules/gvvaughan/lpeg
checksum is the same.